app/views/data_portal/index.html.erb

```diff
@@ -56,11 +56,27 @@
               nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem.
             </p>
           </div>
+          <div class="grid-s-12 grid-m-5 offset-m-1">
+            <div class="datasetimagewrapper">
+              <%= image_tag('data-portal/data-portal-dashboard-1.jpg',
+                :alt => 'Data portal dashboard',
+                :class => 'datasetimage -right')
+              %>
+            </div>
+          </div>
         </div>
       </section>
       <section class="l-dataset -financial-diaries" id="financial-diaries-dataset">
         <div class="row">
           <div class="grid-s-12 grid-m-6">
+            <div class="datasetimagewrapper">
```

Comment:
I've noticed you use this kind of nomenclature for CSS, the one we use is SMACSS (https://smacss.com/), so this class would be `dataset-image-wrapper`.

Reply:
was following the rcss convention for components/elements that you sent the other week